src/modules/billing/mappers/transaction.js

```diff
@@ -240,7 +248,19 @@ const modelToChargeModule = (batch, invoice, invoiceLicence, transaction) => {
     batchNumber: batch.id,
     ...mapChargeElementToChargeModuleTransaction(transaction.chargeElement),
     ...mapLicenceToChargeElementTransaction(invoiceLicence.licence),
-    newLicence: transaction.isNewLicence
+    newLicence: transaction.isNewLicence,
+    calculation: {
```

Comment:
Is this supposed to be here?  This mapper is for mapping data to create the transaction initially in the CM

Reply:
I've removed this. Thanks